docs/languages/cpp.md

```diff
@@ -194,57 +143,77 @@ To go to a symbol's definition, place your cursor on the symbol anywhere its use
 
 ## Debugging
 
+After you have set up the basics of your debugging environment as specified in [Getting Started](#getting_started), you can learn more details about debugging C/C++ here.
+
 VS Code supports the following debuggers for C/C++ depending on the operating system you are using:
 
-* **Linux**: supports debugging using GDB
-* **OS X**: supports using LLDB or GDB
-* **Windows**: currently supports GDB only (using Cygwin or MinGW)
+* **Linux**: using GDB
+* **OS X**: using LLDB or GDB
+* **Windows**: using the Visual Studio Windows Debugger or GDB (using Cygwin or MinGW)
 

```

Comment:
I would drop the "using" and just say "GDB", "LLDB or GDB", "Visual Studio Windows Debugger or GDB (using Cygwin or MinGW)"